### PR TITLE
Aaron troska

### PR DESCRIPTION
**Pull request by Trosaar**
Day one pull. Most component built out. Added assets for icons. working on adding a new comment functionality.

### CODE REVIEW

File: instagram-app/src/Components/CommentSection/CommentSection.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//       )})}
+//       <NewComment />
+//     </div>
+//   );
+// }
+
+class CommentSection extends React.Component {
+  constructor(props){
+    super(props);
+    this.state=props.comments;
```
Comment (ari-mercado) [inline instagram-app/src/Components/CommentSection/CommentSection.js]: State should be set to an object here.

File: instagram-app/src/Components/CommentSection/CommentSection.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  render(props){
+    return(
+      <div>
+        {props.comments.map(comment => { return(
```
Comment (ari-mercado) [inline instagram-app/src/Components/CommentSection/CommentSection.js]: In class components, we need to include the `this` keyword for props.
`this.props`

File: instagram-app/src/Components/PostContainer/PostContainer.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+class PostContainer extends React.Component {
+  constructor(props){
+    super(props);
+    this.state={
+      ...props.eachPost,
+    }
+    console.log(this.state);
+  };
+
```
Comment (ari-mercado) [inline instagram-app/src/Components/PostContainer/PostContainer.js:31]: This click handler brings up a warning message saying. "Do not mutate state directly. Use setState()." Basically it doesn't like the use of *this.state.likes* inside the setState(). A quick fix is to assign a variable to that increments the likes.
`let likes = this.state.likes += 1;
    this.setState({
      likes: likes,
    })`
or using ES6
`let likes = this.state.likes += 1;
    this.setState({ likes })`

**Comment by ari7946** [comment]
## Code Review Checklist

- [x] Follows Lambda Git Flow 
- [x] Frequent, clear, and descriptive commits
- [x] No errors in the console
- [x] Code is written by the student
- [ ] Code is sufficiently commented
- [x] Code is organized and easy to read
- [x] MVP Complete
- [ ] At least 1 stretch completed

### Notes:
Nice work, Aaron. I like the use of the spread operator. Great job on adding the functionality for the likes and the comments.  The app overall could use some CSS and images should include an "alt" prop.

**Pull request merged**